Observations/obs.md

```diff
@@ -0,0 +1,152 @@
+# Observations 
+
+## Overview
+
+An Observation is one single piece of information that is recorded about a person at a moment in time.
+
+Every observation has a Concept as its question, and depending on the datatype of the concept, it has a value that is a number, date, text, Concept, etc.
+
+Most of the information you store in OpenMRS is in the form of Observations, and most Observations happen in an Encounter. When you enter a form in OpenMRS, typically one Encounter is created with anywhere between tens or hundreds of Observations.
+
+Note that an individual Observation is valid only at one moment in time, and it does not carry forward. You may query the system for the last observation for pregnancy status but this does not tell you whether or not the patient is pregnant at any point after the moment of that observation.
+
+Examples of observations include Serum Creatinine of 0.9mg/dL or Review of cardiopulmonary system is normal.
+ 
+## Available operations. 
+
+1. [List observations](#list-observations)
+2. [Create an observation](#create-an-observation)
+3. [Update an observation](#update-an-observation)
+4. [Delete an observation](#delete-an-observation)
+
+### List observations
+
+* #### List all observations.
+    
+    Quickly filter observations with given query parameters. 
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+
+    #### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *patient* | `target_patient_uuid` | patient resource UUID
+    *concept* | `target_concept_uuid`| concept resource UUID (this parameter to be used with patient)
+
+    ```console
+    GET /obs?patient=070f0120-0283-4858-885d-a20d967729cf"
+     ```
+    
+* #### Query observations by UUID.
+
+    Retrieve an observation by its UUID.
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+    
+    ```console
+    GET /obs/:target_observation_uuid
+    ```
+   
+### Create an observation
+
+* To Create an observation you need to specify below attributes in the request body. If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *person* | `Person UUID` | the Person this Obs is acting on.
+    *obsDateTime* | `String` | the time this Obs took place.
+    *concept* | `String` | the coded value/name given to an obs when it is made.
+    *location* | `String` | the location this Obs took place (was taken).
+    *order* | `String` | the order of an Obs.
```

Comment:
`obs.order` is actually a [link to the order](https://docs.openmrs.org/datamodel/openmrs-refapp-data-model-1.9.html#obs.order) (request by a provider) that led to the observation. For example, the doctor orders (aka requests) a lab test and the result of the lab test is stored as an `obs` which is linked to the original order to make it clear that the observation "fulfills" the provider's request.

So, the type for order is actually an `Order UUID`.